### PR TITLE
Added window protocol to external playerView resources

### PR DESCRIPTION
**Pull request by Tanjitsu**
Request secure URL if using secure address.

### CODE REVIEW

File: rcongui/src/components/PlayerView/playerList.js [hunk 1]
```diff
@@ -192,7 +192,7 @@ const PlayerItem = ({
               className={classes.marginRight}
               target="_blank"
               color="inherit"
-              href={`https://steamcommunity.com/profiles/${steamID64}`}
```
Comment (MarechJ) [inline rcongui/src/components/PlayerView/playerList.js:195]: Was this one actually a problem?

Now that I think of it, why don't we put https hardcoded for both. There's no reason to downgrade is there?

Comment (Tanjitsu) [reply rcongui/src/components/PlayerView/playerList.js:195]: No it wasn't causing an issue as it doesn't fetch anything but yeah :)

Comment (MarechJ) [reply rcongui/src/components/PlayerView/playerList.js:195]: Can you hardcode https then? And let merge it :)

Comment (Tanjitsu) [reply rcongui/src/components/PlayerView/playerList.js:195]: Oh ok, yeah ill push it tomorrow !

**Review by MarechJ — APPROVED** [review]

**Pull request merged**